### DISCUSSION TITLE
Unable to push RTSP: (path 'mystream' is not configured)

### DISCUSSION THREAD

**Discussion by hoonkai** [bluenviron/mediamtx#974] [Questions] [answered]
Hi

I'm trying to push a H265 file to the server using `ffmpeg -re -stream_loop -1 -i file.m4v -c copy -f rtsp rtsp://localhost:8554/mystream` but I keep getting the error:

```
2022/06/07 23:54:59 INF [RTSP] [conn 127.0.0.1:55855] closed (path 'mystream' is not configured)
2022/06/07 23:54:59 INF [RTSP] [session 801883554] closed (not in use)
```

And FFmpeg gives the following error:
```
Could not write header for output file #0 (incorrect codec parameters ?): Server returned 400 Bad Request
```

Am I supposed to specify `mystream` in the yaml file? The doc on the front page under "Basic usage" has no mention of it though.

Any advice appreciated.

**Comment by hoonkai** [comment, marked as answer]
Sorted. I had to add a path in the yaml.

**Reply by hanskirch** [reply]
I have been attempting to add a path but have had no success. Would you mind sharing what you added and also to what part of the file?

**Reply by AdrianGarside** [reply]
Same here. Wish they'd given a more complete self-answer :(

**Reply by AdrianGarside** [reply]
Aha, looking at the log output, I determined the default stream name is 'cam'. I still don't see this documented but it's clear to me now that it's coming from:
```
paths:
  cam:
    source: rpiCamera
```
So to change the name, change 'cam:' to 'somethingelse:'

[a comment by an automated account is not shown]
